fdbserver/TagPartitionedLogSystem.actor.cpp

```diff
@@ -117,12 +217,47 @@ struct TagPartitionedLogSystem : ILogSystem, ReferenceCounted<TagPartitionedLogS
 		return dbgid;
 	}
 
+	void addPseudoLocality(int8_t locality) override {
```

Comment:
These two functions (addPseudoLocality and getPseudoPopTag) do not need to be virtual functions, since they have no external users

Reply:
Done for addPseudoLocality(). getPseudoPopTag() is used in log router, so it stays as virtual.